tests/baselines/reference/mixinAccessModifiers.errors.txt

```diff
@@ -181,16 +183,18 @@ tests/cases/conformance/classes/mixinAccessModifiers.ts(130,4): error TS2445: Pr
     function f8(x: ProtectedGeneric<{a: void;}> & ProtectedGeneric2<{a:void;b:void;}>) {
     	x.privateMethod(); // Error, private constituent makes method inaccessible
     	  ~~~~~~~~~~~~~
-!!! error TS2546: Property 'privateMethod' has conflicting declarations and is inaccessible in type 'ProtectedGeneric<{ a: void; }> & ProtectedGeneric2<{ a: void; b: void; }>'.
+!!! error TS2339: Property 'privateMethod' does not exist on type 'never'.
+!!! error TS2339:   The intersection 'ProtectedGeneric<{ a: void; }> & ProtectedGeneric2<{ a: void; b: void; }>' was reduced to 'never' because property 'privateMethod' exists in multiple constituents and is private in some.
     	x.protectedMethod(); // Error, protected when all constituents are protected
     	  ~~~~~~~~~~~~~~~
-!!! error TS2445: Property 'protectedMethod' is protected and only accessible within class 'ProtectedGeneric<{ a: void; }> & ProtectedGeneric2<{ a: void; b: void; }>' and its subclasses.
+!!! error TS2339: Property 'protectedMethod' does not exist on type 'never'.
+!!! error TS2339:   The intersection 'ProtectedGeneric<{ a: void; }> & ProtectedGeneric2<{ a: void; b: void; }>' was reduced to 'never' because property 'privateMethod' exists in multiple constituents and is private in some.
     }
     
     function f9(x: ProtectedGeneric<{a: void;}> & ProtectedGeneric<{a:void;b:void;}>) {
     	x.privateMethod(); // Error, private constituent makes method inaccessible
     	  ~~~~~~~~~~~~~
-!!! error TS2546: Property 'privateMethod' has conflicting declarations and is inaccessible in type 'ProtectedGeneric<{ a: void; }> & ProtectedGeneric<{ a: void; b: void; }>'.
+!!! error TS2341: Property 'privateMethod' is private and only accessible within class 'ProtectedGeneric<T>'.

```

Comment:
? What happened here?

Reply:
This is an intended change. Error `TS2546` is gone and has been superseded by the reduction of intersections with conflicting privates. In this particular case, however, the error wasn't even correct because there is only one declaration of `privateMethod` in the intersection (and therefore the intersection didn't get reduced to `never`).

Reply:
Huh, ok.